lib/data_hygiene/policy_tagger.rb

```diff
@@ -0,0 +1,80 @@
+class PolicyTagger
+  def self.process_from_csv(csv_location, logger: Logger.new(nil))
+    CSV.foreach(csv_location, headers: true) do |row|
+      policies_to_remove = row.fetch("policies_to_remove") ? policies_to_remove.split(" ") : []
```

Comment:
Looking at the example data this makes much more sense now. Sorry for the confusion!

You could still remove the argument to `split` as that's the default. Otherwise :+1: